### DISCUSSION TITLE
Foliumap to_gradio() not working

### DISCUSSION THREAD

**Discussion by be84** [opengeos/leafmap#748] [Q&A] [answered]
Hello everyone,

I hope this message finds you well. I am currently experiencing an issue when trying to replicate the example of using Gradio with Leafmap. Unfortunately, no map is shown.

Issue Description:
I'm using leafmap version 0.32.1 and gradio version 4.36.0. Interestingly, the same code works on a Colab notebook, but not on Windows or WSL.

Install leafmap version 0.32.1 and gradio version 4.36.0.
Run the following code:

Expected Behavior:
A map should be displayed within the Gradio interface.

Actual Behavior:
No map is shown in the Gradio interface when running on Windows or WSL.

Additional Information:

The code works correctly on a Colab notebook with the same package versions.
The issue seems specific to the environment (Windows and WSL).
Any assistance or insights would be greatly appreciated. Thank you!

Steps to Reproduce:
```
  import leafmap.foliumap as lf
  import gradio as gr
  
  def generate_map():
    m = lf.Map()
    return m.to_gradio(500, 500)
  
  with gr.Blocks() as demo:
    gr.Markdown('Test')
    block = gr.HTML(generate_map())
  demo.launch()

**Comment by giswqs** [comment, marked as answer]
I am no longer using gradio actively. I would recommend using [solara](https://solara.dev/), which is much better for mapping applications with bidirectional functionality. See example at https://github.com/opengeos/solara-geospatial
